### ISSUE TITLE
Build from source docker container fails due to node version mismatch

### ISSUE THREAD

**Issue by TheRealMephisto** [VSCodium/vscodium#2011]
**Describe the bug**
When following the build process as described in the [documentation](https://github.com/VSCodium/vscodium/blob/master/docs/howto-build.md#build-docker), the build fails due to an "incompatible module" error.
The module `@playwright/test@1.45.0` expects a version >=18 for node, but version 16.20.2 is used in the docker container.

Here is the relevant output of the execution of `./build/build.sh`:
```Error message
yarn install v1.22.19
$ node build/npm/preinstall.js
[1/4] Resolving packages...
[2/4] Fetching packages...
error @playwright/test@1.45.0: The engine "node" is incompatible with this module. Expected version ">=18". Got "16.20.2"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/docs/index.md) and [the Troubleshooting page](https://github.com/VSCodium/vscodium/blob/master/docs/troubleshooting.md) and my issue is not mentioned there.

**To Reproduce**
1. Go to [the mentioned step in the documentation](https://github.com/VSCodium/vscodium/blob/master/docs/howto-build.md#build-in-docker) for comparison (detailed steps are reported in here as well):
2. Create the container (I used the x64 variant)
3. Inside the container, execute the steps
```Execute build
cd /root/vscodium
./build/build.sh
```
4. Wait for a few first steps to run trhough until `$ node build/npm/preinstall.js` is exectuted
5. See error

**Expected behavior**
Build process should run through without stopping due to an error

**Screenshots**
Not necessary, relevant output of the build.sh was pasted in the beginning of this report.

**Desktop (please complete the following information):**
 - OS: Manjaro
 - Architecture: x64
 - Version: 6.6.46-1
 - App Manager: pamac
 - Sandboxed: no

**Additional context**
1. I decided to include the information on my Desktop even though I do not expect this information to be relevant for the problem at hand.
2. First attempt of fixing the issue
Furthermore, I've already tried replacing the nodejs version installed in the docker container by running 
```shell
sudo apt-get remove nodejs
```
and setting up the desired node version for installer
```shell
$ curl -fsSL https://deb.nodesource.com/setup_[VN].x | sudo -E bash -
```
*(copied from the build/build_docker.sh script, with adjusted versions)*
Tried that for [VN]=18 as well as [VN]=22.

In both cases, the subsequent run of
```
sudo apt-get install nodejs
```
revealed that the solution is not that simple:
```output
The following packages have unmet dependencies:
 nodejs : Depends: libc6 (>= 2.28) but 2.27-3ubuntu1.6 is to be installed
E: Unable to correct problems, you have held broken packages.

```
3. Will come back to this issue later myself.
Of course I do appreciate if someone commits a quick fix in the meantime!

**Comment by daiyam** [comment]
That section (docker) was for an older version... I removed it.

**Comment by TheRealMephisto** [comment]
@daiyam So it's not possible to build using a docker container?

**Comment by TheRealMephisto** [comment]
I did some further digging and found a more [up to date version](https://hub.docker.com/r/vscodium/vscodium-linux-build-agent/tags?page=&page_size=&ordering=&name=) of the docker image.
So far it is compiling without problems.

I'll report later whether it compiled successfully and if so, I'll create a pull request for an updated documentation.

**Comment by TheRealMephisto** [comment]
I'll do that. 

Even though I found the idea of having a prebuilt docker image available quite appealing, even with [the version focal-x64](https://hub.docker.com/r/vscodium/vscodium-linux-build-agent), I ran into an error.

This error was during the compilation process though, but I'm currently unable and unwilling to invest the time in understanding its origin.

Will attach the relevant output as a [snippet file](https://github.com/user-attachments/files/16801354/build_failure_docker_vscodium-build-agent.txt) to this comment, in case somebody cares to look into it.

I'll close this issue as it can be considered resolved after removal of the corresponding part of the documentation.
